templates/blocks/blok_dossier.twig

```diff
@@ -1,38 +1,45 @@
+                />
+                <div class="absolute inset-x-0 bottom-0 h-32 max-h-full bg-gradient-to-t from-gray-800"></div>
+            </div>
+        </div>
+    </div>
     <div class="w-full mx-auto max-w-960">
         <div class="lg:grid grid-cols-3">
             <div class="lg:col-span-2 lg:col-start-2 relative z-20">
                 <div class="px-4 pt-4">
-                    {% if block.tekst_boven_dossier %}
-                        <h1 class="text-3xl font-bold">{{ block.tekst_boven_dossier }}</h1>
-                    {% else %}
-                        <h1 class="font-bold">
-                            <span class="block text-sm tracking-wide uppercase font-round">Dossier</span>
+                    <h1 class="font-bold">
+                        <span class="rounded-md px-2 py-0.5 text-sm tracking-wide uppercase font-black font-round bg-blauw text-white">📂 Dossier</span>
+                        {% if block.tekst_boven_dossier %}
+                            <span class="block text-3xl">{{ block.tekst_boven_dossier }}</span>
+                        {% else %}
                             <span class="block text-3xl">{{ block.term.name }}</span>
-                        </h1>
-                    {% endif %}
-                    <div class="prose text-white py-4">{{ block.term.description }}</div>
+                        {% endif %}
+                    </h1>
+                    <div class="hidden lg:block prose font-bold font-round text-gray-400 py-4">Alles over de gemeente&shy;raadse&shy;verkiezingen 2022 van Roosendaal, Bergen op Zoom en Woensdrecht in ZuidWest Kiest.</div>
                 </div>
-
                 <ul class="divide-y divide-gray-700 mb-4">
-                    {% for post in block.posts %}
-                        {% include 'partial/dossier-post-tile.twig' with { post: post } %}
-                    {% endfor %}
+                    {% for post in block.posts %} {% include 'partial/dossier-post-tile.twig' with { post: post } %} {% endfor %}
                 </ul>
-                <div class="block px-2 pb-4 hover:underline ">
-                    <a href="{{ block.term.link }}" class="p-2">Alles over {{ block.term.name }} →</a>
+                <div class="block px-2 pb-4">
+                    <a href="{{ block.term.link }}" class="text-blauw hover:bg-gray-700 block px-2 mx-1 my-0.5 rounded">→ Alles over {{ block.term.name }}</a>
                 </div>
             </div>
-
             <div class="col-start-1 row-start-1 relative z-10">
                 <div class="absolute inset-y-0 -inset-x-8">
-                    <img class="absolute inset-0 w-full h-full object-cover" loading="lazy"
-                         src="{{ Image(block.term.meta('dossier_afbeelding_breed')) }}"/>
+                    <img class="absolute inset-0 w-full h-full object-cover" loading="lazy" src="{{ Image(block.term.meta('dossier_afbeelding_hoog')) }}" />
```

Comment:
Maakt mij niet uit of je `block.term.dossier_afbeelding_hoog` of `block.term.meta('dossier_afbeelding_hoog')` gebruikt, als je er maar één gebruikt. Also, deze maat wordt niet door thumbor gehaald; is dat bewust gekozen?

Reply:
Oke, en nee wilde wachten met Thumbor tot de juiste syntax